zap/src/main/java/org/parosproxy/paros/core/scanner/HostProcess.java

```diff
@@ -640,6 +657,17 @@ private boolean canScanNode(StructuralNode node) {
             }
             return false;
         }
+        FilterResult filterResult = filterNode(node);
+        if (filterResult.isFiltered()) {
+            if (log.isDebugEnabled()) {
+                log.debug(
+                        "Ignoring filtered node: "
+                                + node.getName()
+                                + " Reason for Ignoring: "
+                                + filterResult.toString());
```

Comment:
btw, this can just use the reason (the filtered state and additional wording is not needed).

Reply:
sure.

Reply:
@thc202 , corrected the comment little bit. as we need Node name so didn't removed that.

Reply:
Sorry, I was suggesting using `getReason()` instead of `toString()`, the latter includes more than needed. (New wording looks good.)

Reply:
oh .. yeah that is correct.

Reply:
@thc202  done.